common/sketch/src/main/java/org/apache/spark/util/sketch/Murmur3_x86_32.java

```diff
@@ -60,6 +60,8 @@ public static int hashUnsafeWords(Object base, long offset, int lengthInBytes, i
   }
 
   public static int hashUnsafeBytes(Object base, long offset, int lengthInBytes, int seed) {
+    // This is not compatible with original and another implementations.
+    // But remain it for backward compatibility for the components existing before 2.3.
```

Comment:
NIT: touch this up. The sentence is a bit weird, i.e.: `However we retain this implementation for backwards compatibility with pre-existing (pre 2.3) components.`

Reply:
Will correct it in the follow-up PR.